python/rikai/spark/sql/codegen/pytorch.py

```diff
@@ -41,7 +41,7 @@ def move_tensor_to_device(data, device):
     return data
 
 
-def generate_udf(payload: ModelSpec):
+def generate(payload: ModelSpec, is_udf: bool = True):
```

Comment:
Use is_udf = False to generate the function which can be used without PySpark

only for pytorch now, need feedbacks from reviewers and then apply to other codegen